tests/scripts/run_embedded_app.py

```diff
@@ -40,16 +40,35 @@ def launch_app(args):
         init_msg = "The app is initialized" if is_initialized() else "The app is not initialized"
         print(init_msg)
 
-    app = pymechanical.App(
-        version=int(args.version),
-        private_appdata=args.private_appdata,
-        copy_profile=False,
-        globals=globals(),
-    )
+    if args.update_globals_kwarg:
+        app = pymechanical.App(
+            version=int(args.version),
+            private_appdata=args.private_appdata,
+            copy_profile=False,
+            globals=globals(),
+        )
+    else:
+        app = pymechanical.App(
+            version=int(args.version),
+            private_appdata=args.private_appdata,
+            copy_profile=False,
+        )
+        app.update_globals(globals())
 
     return app
 
 
+def test_globals(args):
+    """Test ViewOrientationType doesn't throw an error when globals are updated."""
+    if args.build_gallery_flag:
```

Comment:
the only test coverage we were missing was when BUILDING_GALLERY was set to True and the second app instance had globals updated.